src/vcpkg/spdx.cpp

```diff
@@ -34,36 +117,12 @@ static void append_move_if_exists_and_array(Json::Array& out, Json::Object& obj,
     }
 }
 
-static StringView find_cmake_invocation(StringView contents, StringView command)
-{
-    auto it = Strings::case_insensitive_ascii_search(contents, command);
-    if (it == contents.end()) return {};
-    it += command.size();
-    if (it == contents.end()) return {};
-    if (ParserBase::is_word_char(*it)) return {};
```

Comment:
@ras0219-msft I really don't understand the significance of these 'word char' checks here. If a file has `vcpkg_from_github .... vcpkg_from_git`, then we should skip over the first one and go to the real one, which I added a test for and fixed. Please double check that I did not misunderstand these' purpose.